srcpkgs/cnrdrvcups-sfp/template

```diff
@@ -0,0 +1,219 @@
+	ln -s libColorGearCsfp.so.2.0.0 libColorGearCsfp.so.2
+	ln -s libColorGearCsfp.so.2.0.0 libColorGearCsfp.so
+
+	popd
+
+	vsed -e 's/cnsetuputil2l\/images\/128x128/icons\/hicolor\/128x128\/apps/' -i cnrdrvcups-utility-5.00/data/cnsetuputil2l.desktop
+	vsed -e 's/Application;Utility/Utility/' -i cnrdrvcups-utility-5.00/data/cnsetuputil2l.desktop
+}
+
+do_install() {
+	for _subdir in "buftool" "cngplp" "cnjbig" "rasterfilter"
+	do
+		( cd ${_common_dir}/${_subdir} && make install DESTDIR=${DESTDIR} COMMON_SUFFIX=2 )
+	done
+
+	for _subdir in "cngplp/files" "cngplp" "cpca" "ppd" "StatusMonitor"
+	do
+		( cd ${_driver_dir}/${_subdir} && make install DESTDIR=${DESTDIR} )
+	done
+
+	pushd "lib/libs${_bits}"
+
+	vbin cnpkmodulencapr
+	vbin cnrsdrvsfp
+	vbin cnsetuputil2l
+
+	vmkdir usr/lib/Canon/CUPS_SFPR/Bidi
+	for _so_file in \
+		libcaio_usb_cdcr.so \
+		libccpd_utilr.so \
+		libcomm_ncapcaior.so \
+		libcomm_stdoutr.so \
+		libcomm_usbmlportr.so \
+		libcomm_usbsockr.so \
+		libcomm_websrvr.so
+	do
+		vcopy ${_so_file}* usr/lib/Canon/CUPS_SFPR/Bidi
+	done
+
+	vmkdir usr/lib/Canon/CUPS_SFPR/Bins
+	vinstall commandfilefilterr 755 usr/lib/Canon/CUPS_SFPR/Bins
+
+	vmkdir usr/lib/Canon/CUPS_SFPR/Libs
+	for _so_file in \
+		libCUPS_Communicatorr.so \
+		libCommIsolationr.so \
+		libcanon_commonr.so \
+		libinfo_analyzer.so \
+		libinfor.so
+	do
+		vcopy ${_so_file}* usr/lib/Canon/CUPS_SFPR/Libs
+	done
+
+	for _so_file in \
+		libColorGearCsfp.so \
+		libcaepcmsfp.so \
+		libcanon_slimsfp.so \
+		libcanonncapr.so \
+		libcnncapcmr.so \
+		libncapfilterr.so
+	do
+		vcopy ${_so_file}* usr/lib
+	done
+
+	vmkdir usr/lib/Canon/CUPS_SFPR/Utilities
+	vinstall pksmncapr 4755 usr/lib/Canon/CUPS_SFPR/Utilities
+
+	for _locale in "de" "es" "fr" "it" "ja" "ko" "zh_CN" "zh_TW"
+	do
+		vmkdir usr/share/locale/${_locale}/LC_MESSAGES
+		vinstall ${_locale}/LC_MESSAGES/cnsetuputil2l.mo 644 usr/share/locale/${_locale}/LC_MESSAGES
+	done
+
+	vmkdir usr/share/caepcm/sfp
+	vcopy ../data/sfp/* usr/share/caepcm/sfp
+
+	vmkdir usr/share/ncapfilterr
+	vinstall ThLB_37A.BIN 644 usr/share/ncapfilterr
+
+	popd
+
+	vmkdir usr/lib/udev/rules.d
+	vinstall ${_driver_dir}/rules/80-usb-ncapstatusui2l.rules 644 usr/lib/udev/rules.d
+
+	vmkdir usr/share/cups/usb
+	vinstall ${_driver_dir}/rules/canon-sfp-printerr.usb-quirks 644 usr/share/cups/usb
+
+	vcopy ${_driver_dir}/StatusMonitor/Message usr/lib/Canon/CUPS_SFPR/Utilities
+
+	vmkdir usr/share/applications
+	vinstall cnrdrvcups-utility-5.00/data/cnsetuputil2l.desktop 644 usr/share/applications
+
+	vmkdir usr/share/icons/hicolor/128x128/apps
+	vinstall cnrdrvcups-utility-5.00/data/cnsetuputil.png 644 usr/share/icons/hicolor/128x128/apps
+
+	vlicense ../linux-UFRIILT-drv-v500-uken/Documents/LICENSE-EN.txt LICENSE
+
+	vlicense ${_common_dir}/cngplp/LICENSE.canon.txt LICENSE.cngplp
+	vlicense ${_driver_dir}/StatusMonitor/LICENSE.canon.txt LICENSE.statusmonitor
+	vlicense ${_driver_dir}/cpca/cnpklib/LICENSE.canon.txt LICENSE.cpca.cnpklib
```

Comment:
These may be a bit messed up, some of them refer to GPL and probably aren't needed.